WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -27,7 +27,6 @@
         </Grid.RowDefinitions>
 
         <TextBlock
-            AutomationProperties.HeadingLevel="Level1"
```

Comment:
fix build break from yesterday:
https://github.com/microsoft/WinUI-Gallery/commit/913286a437b6ad3e4d74a493e7a08a5629807963

Reply:
main should have the latest build fixes now.